### PR TITLE
Added Danske Bank SE DABASESX

### PR DESCRIPTION
**Pull request by janeklof**

### CODE REVIEW

File: docs/banks/DABASESX.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+id: dabasesx
+title: Danske Bank (DABASESX)
+sidebar_label: Danske Bank
```
Comment (henrikwidlund) [inline docs/banks/DABASESX.md:4]: Should this be Danske Bank Sweden?

File: docs/banks/DABASESX.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+---
+
+## Status Highlights
+
+| Status | Product | Comment |
+|:---|---|---|
+|![](https://img.shields.io/badge/status-important-important.svg)| PIS | Sandbox doesn't support specifying debtor account for payment, this is therefore omitted by us in the request to the ASPSP. The PSU must choose which account that should be debited in the ASPSP's SCA flow. |
```
Comment (henrikwidlund) [inline docs/banks/DABASESX.md:11]: Might not be needed to be so specific, could it be enough to say that the PSU must select the account that should be debited regardless of what is sent in the Payment Initiation request in sandbox?

File: docs/banks/DABASESX.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+|----------------|----------|--------------|--------------|--------------|
+|Sandbox         |OAuth Redirect | User/password credentials  | ![](https://img.shields.io/badge/status-active-success.svg) | Please see `Sandbox Test Data` |
+|Production      |OAuth Redirect | Mobilt BankID | ![](https://img.shields.io/badge/status-active-success.svg) | When choosing debtor account for payment in the banks SCA webpages, a warning message "kontonummerverifikation misslyckades" is presented by the bank. It is unknown what this means and it has been reported to the bank, but does not seem to have any impact on the payment authorisation. |
+
+### Sandbox Test Data
+Credentials for SCA:
+- Username: 8195475386
+- Password: xUKSWPgHy2H2XBt8cv
+
+Sandbox only supports British accounts, this applies both to AIS and PIS, currency is therefore `GBP` and IBANs are for GB as well. We are mapping UK.OBIE.SortCodeAccountNumber to BBAN in sandbox to make it more aligned with the rest of the system.
```
Comment (henrikwidlund) [inline docs/banks/DABASESX.md:25]:
```suggestion
Sandbox only supports British accounts, this applies both to AIS and PIS, currency is therefore `GBP` and IBANs are for GB as well. We are mapping UK.OBIE.SortCodeAccountNumber to BBAN in sandbox to make it more aligned with other Swedish ASPSPs.
```

File: website/i18n/en.json [hunk 4]
```diff
@@ -21,6 +21,10 @@
         "title": "Supported Banks",
         "sidebar_label": "Supported banks"
       },
+      "banks/dabasesx": {
+        "title": "Danske Bank (DABASESX)",
+        "sidebar_label": "Danske Bank"
```
Comment (henrikwidlund) [inline website/i18n/en.json:26]: Same here, might want to clarify that it is the Swedish Danske Bank.

File: docs/banks/DABASESX.md (hunk 1 above)
Comment (janeklof) [reply docs/banks/DABASESX.md:4]: Well, it is listed in the Swedish section and I looked at their naming on their Swedish pages and looked like they only use "Danske Bank".. Bolagsverket says: "Danske Bank A/S, Danmark, Sverige Filial"

Comment (henrikwidlund) [reply docs/banks/DABASESX.md:4]: Cool, keep it as it is then :)

**Pull request merged**